packages/mobile/src/screens/UsernameTaken/UsernameTaken.screen.tsx

```diff
@@ -40,6 +42,16 @@ const UsernameTakenScreen: React.FC<UsernameTakenScreenProps> = () => {
     )
   }
 
+  useEffect(() => {
```

Comment:
I know it works but it doesn't look right. We should identify the cause this view is on top in the first place